src/app/study-filters/study-filters.component.ts

```diff
@@ -22,7 +22,7 @@ export class StudyFiltersComponent extends StatefulComponent implements OnInit,
 
   @Validate(SetNotEmpty, {message: 'Select at least one.'})
   public selectedStudies: Set<string> = new Set<string>(['']); // Used by the child component(Treelist checkbox)
-  public allStudyNodes: DatasetNode[] = null;
+  public allStudyNodes: DatasetNode = null;
```

Comment:
I think renaming `allStudyNodes` to `rootDataset` or something similar should be more accurate.